### PR TITLE
[SDCI] add message about gitlab test settings potentially failing

### PR DESCRIPTION
**Pull request by alxch-**
### What does this PR do? What is the motivation?

We had reports from customers that the validation through "Test settings" was failing but everything was working after saving. I prefer to add this to the documentation in case customers stop the setup because of it.

### Merge instructions

- [X] Please merge after reviewing

### CODE REVIEW

File: content/en/continuous_integration/pipelines/gitlab.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
+
```
Comment (estherk15) [inline content/en/continuous_integration/pipelines/gitlab.md:2]:
```suggestion
```

File: content/en/continuous_integration/pipelines/gitlab.md [hunk 2]
```diff
@@ -157,7 +158,7 @@ Fill in the integration configuration settings:
 **Default**: (empty, no additional tags)<br/>
 **Note**: Available only in GitLab.com and GitLab >= 14.8 self-hosted.
 
-You can test the integration with the **Test settings** button (only available when configuring the integration on a project). After it's successful, click **Save changes** to finish the integration set up.
+You can test the integration with the **Test settings** button (only available when configuring the integration on a project). After it's successful, click **Save changes** to finish the integration set up. Some customers have reported that the **Test settings** button sometimes fails. In this case, click **Save changes** and check that the first webhooks that are sent are successful by looking at the history in the "Recent events" section below.
```
Comment (estherk15) [inline content/en/continuous_integration/pipelines/gitlab.md:161]: What is the "Recent events" section? Is this a reference to a section on this page?

```suggestion
You can test the integration with the **Test settings** button (only available when configuring the integration on a project). After it's successful, click **Save changes** to finish the integration set up. If the button fails, click **Save changes** and check that the first webhooks sent are successful by looking at the history in the "Recent events" section below.
```

Comment (alxch-) [reply content/en/continuous_integration/pipelines/gitlab.md:161]: Yes, on the same page there is a "Recent events" section
image

**Comment by alxch-** [comment]
@estherk15 this is ready to be merged :) Thanks for the review

**Review by estherk15 — APPROVED** [review]

**Pull request merged**